command/rm.go

```diff
@@ -35,13 +36,20 @@ Examples:
 
 	4. Delete all matching objects and a specific object
 		 > s5cmd {{.HelpName}} s3://bucketname/prefix/* s3://bucketname/object1.gz
+	
+	5. Delete all matching objects but exclude the ones with .txt extension or starts with "main"
```

Comment:
Pattern in usage text is only quoted here, it should be consistent around other usage texts IMO.

Reply:
change to `starting`. I saw double quotes are used in `cp.go`, therefore I added the double quotes but I can change it for sure.

Reply:
I was just referring exclude pattern strings. I think it does not matter whether we quote them or not since they do not contain whitespaces, I leave it yo your taste.